src/main/java/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,23 @@
+package cinema.controller;
+
+import cinema.model.dto.UserRequestDto;
+import cinema.security.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto userRequestDto) {
```

Comment:
```suggestion
    public void register(@RequestBody UserRequestDto userRequestDto) {
```
method names should be verbs